plugins/Rockets/RocketsPlugin.cpp

```diff
@@ -977,15 +972,6 @@ class RocketsPlugin::Impl : public ActionInterface
                               "renderer");
     }
 
-    void _handleFrameBuffer()
-    {
-        // don't add framebuffer to websockets for performance
-        using namespace rockets::http;
-        _rocketsServer->handleGET(ENDPOINT_FRAME_BUFFERS,
-                                  _engine.getFrameBuffer());
-        _handleObjectSchema(ENDPOINT_FRAME_BUFFERS, _engine.getFrameBuffer());
```

Comment:
This was used in the past to get the depth and color buffers in a Python notebook and do stuff with them for the reprojection prototype. Would it be difficult to change this end point to return a list of buffer names and have an RPC to retrieve a frame buffer by name. Doesn't have to go in this PR.

Reply:
Remove json serialization for FrameBuffer if this is removed.